### ISSUE TITLE
azurerm_cosmosdb_sql_container needs throughput and TTL settings

### ISSUE THREAD

**Issue by ukphillips** [terraform-providers/terraform-provider-azurerm#4802]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The new azurerm_cosmosdb_sql_container in 1.34 is a welcome addition to the provider - but lacking in some key areas to make it viable for managing real infrastructure. Currently it does not include throughput settings or TTL configuration. Both of these are vital to managing cost and performance of a production instance of cosmos. 

### New or Affected Resource(s)

azurerm_cosmosdb_sql_container

### Potential Terraform Configuration

```
resource "azurerm_cosmosdb_sql_container" "example" {
  name                = "example-container"
  resource_group_name = "${azurerm_cosmosdb_account.example.resource_group_name}"
  account_name        = "${azurerm_cosmosdb_account.example.name}"
  database_name       = "${azurerm_cosmosdb_sql_database.example.name}"
  partition_key_path  = "/definition/id"
  ttl                            = "1000"
  collection_throughput = "1000"
  
  unique_key {
    paths = ["/definition/idlong", "/definition/idshort"]
  }
}
```

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References
https://github.com/terraform-providers/terraform-provider-azurerm/issues/548

**Comment by katbyte** [comment]
related: #4620 #4309 #3623 

would be fixed by #4616

**Comment by EDasilvaCorreia** [comment]
hi all, any news on this?

**Comment by ElvenSpellmaker** [comment]
I just bumped into this, it seems you can't specify the throughput at the DB level either. 🤔

**Comment by ElvenSpellmaker** [comment]
It seems #5203 will add this feature to both DB and Containers, we'll have to await a release 🙂

**Comment by tombuildsstuff** [comment]
Fixed via #5203 which will ship as a part of v1.40 - thanks for the ping @ElvenSpellmaker

**Comment by antempus** [comment]
Really glad this got merged in, this will help us move to using terraform for container management. 

Will this now allow for non-destructive changes to the throughput/ttl on apply?
